include/deepstream/core/fwd.hpp

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2017 deepstreamHub GmbH
```

Comment:
The style guide needs applying to a large part of the project as well. I'll make a note to remove these when I open a formatting-only PR.